src/main/java/controller/BitPay.java

```diff
@@ -33,13 +33,14 @@
 import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.bitcoin.core.ECKey;
+import org.bitcoinj.core.ECKey;
 
 public class BitPay {
 
     private static final String BITPAY_API_VERSION = "2.0.0";
     private static final String BITPAY_PLUGIN_INFO = "BitPay Java Client " + BITPAY_API_VERSION;
-    private static final String BITPAY_URL = "https://bitpay.com/";
+    public static final String BITPAY_URL = "https://bitpay.com/";
```

Comment:
why turn this into a public string? I don't see this change being leveraged anywhere? It is a super small thing, but maybe I am missing something?